projects/control-service/projects/pipelines_control_service/src/integration-test/resources/application-private-builder.properties

```diff
@@ -1,3 +1,6 @@
 datajobs.builder.registrySecret=integration-test-docker-pull-secret
 datajobs.builder.registrySecret.content.testOnly=${BUILDER_TEST_REGISTRY_SECRET}
 datajobs.builder.image=${DOCKER_REGISTRY_URL}/versatiledatakit/job-builder-prerelease:1.2.3
+datajobs.deployment.dataJobBaseImage=ghcr.io/murphp15/versatiledatakit/data-job-base-python-3.7:latest
```

Comment:
We have a versatile-data-kit-dev account . Why not use it?

Reply:
because that is used to store the built image. 
I need the base image to come from a different repo to confirm the auth is working correctly. 
The base image and the save image being in the same private registry is actually supported at the moment. 



I actually just submitted this in the last commit to test it. 
I was going to clean this up before merging. 
I will move the PR back into draft .

Reply:
I will address this before merging.